src/components/FishTankSwitch.vue

```diff
@@ -1,20 +1,20 @@
 <template>
   <div  
-    class="switch">
+    :class="$style.switch">
     <label 
       :for="(id !==null? id: labelId)"
-      class="input-checkbox-label">
+      :class="$style.inputCheckboxLabel">
```

Comment:
Lets try to minify the class names a bit. Lets change "inputCheckboxLabel" to "label". Simplify and keep the class names closer to their purpose.